spec/models/solidus_paypal_commerce_platform/paypal_order_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe SolidusPaypalCommercePlatform::PaypalOrder, type: :model do
+  describe '#to_json' do
+    subject(:to_json) { described_class.new(order).to_json('intent') }
+
+    let(:order) { create(:order_ready_to_complete) }
+
+    it { expect { to_json }.not_to raise_error }
+
+    if Spree.solidus_gem_version >= Gem::Version.new(2.11)
```

Comment:
Couldn't we use two contexts with different `Spree.solidus_gem_version`?
This way the whole functionality would always be tested, and not dependently of the bundle when running the tests.

Reply:
If I had made two contexts would mean that I have had stub the Spree.solidus_gem_version, in that way I can test the method as BlackBox.